examples/rust/objectron/src/objectron.rs

```diff
@@ -97,17 +97,7 @@ pub struct AvDepthData {
 /// Nested message and enum types in `AVDepthData`.
 pub mod av_depth_data {
     /// Indicates the general accuracy of the depth_data_map.
-    #[derive(
-        Clone,
-        Copy,
-        Debug,
-        PartialEq,
-        Eq,
-        Hash,
-        PartialOrd,
-        Ord,
-        ::prost::Enumeration
-    )]
+    #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, ::prost::Enumeration)]
```

Comment:
What happened to this file? It is auto-generated - did you accidentally cargo-fmt it?

Reply:
Also, we should mark it as auto-genned, like we do for the other files

Reply:
oh wasn't aware it's auto-gened. Yes I touched it, should be left unchanged then
Looking after adding a marker!